checkov/terraform/checks/resource/azure/AzureManagedDiskEncryption.py

```diff
@@ -16,8 +16,11 @@ def scan_resource_conf(self, conf):
             return CheckResult.PASSED
         if 'encryption_settings' in conf:
             if isinstance(conf['encryption_settings'][0], dict):
-                self.evaluated_keys = ['encryption_settings]/[0]/[enabled]']
-                return CheckResult.PASSED if conf['encryption_settings'][0]['enabled'][0] else CheckResult.FAILED
+                self.evaluated_keys = ['encryption_settings]/[0]/enabled']
```

Comment:
```suggestion
                self.evaluated_keys = ['encryption_settings/[0]/enabled']
```
can you also remove that square bracket